nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -1798,6 +1798,29 @@
           configuration.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The option
+          <link linkend="opt-services.xserver.desktopManager.runXdgAutostartIfNone">services.xserver.desktopManager.runXdgAutostartIfNone</link>
+          was added in order to automatically run XDG autostart files
+          for sessions without a desktop manager. This replaces helpers
+          like the <literal>dex</literal> package.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          When setting
+          <link linkend="opt-i18n.inputMethod.enabled">i18n.inputMethod.enabled</link>
+          to <literal>fcitx5</literal>, it no longer creates
+          coresponding systemd user services. It now relies on XDG
+          autostart files to start and work properly in your desktop
+          seesions. If your are using only a window manager without a
```

Comment:
```suggestion
          sessions. If your are using only a window manager without a
```

Reply:
Sorry @oxalica. Can you also fix those typos?